playwright/tests/page-objects/inscription.ts

```diff
@@ -111,6 +115,86 @@ export class Collapsible {
   }
 }
 
+export class Badge {
```

Comment:
IMO as written before, this should not exist. 
It should be included in the input page object.

As far as I understand it, now there is no input component (in the properties section) which does not have this badge overlay.
So then the input page object should handle all the stuff around this. how it can write to it, how stuff can be validated. 
E.g. there was a good example for this on the text component test. Why should the test writer care about the knowledge that there is a badge textarea. all he want to test is a textarea, but now he has to write `general.badge({ label: 'Content', textarea: true });`. That's not the idea of a page object, the page object should hide as much of the implementation as possible and needed so the test code gets simpler an more reusable. even if this means that code in the page object gets duplicated...